api/src/main/java/org/openmrs/module/rwandahivflowsheet/utils/Utils.java

```diff
@@ -0,0 +1,18 @@
+package org.openmrs.module.rwandahivflowsheet.utils;
+
+import org.openmrs.Concept;
+import org.openmrs.DrugOrder;
+import org.openmrs.Order;
+import org.openmrs.api.context.Context;
+
+public class Utils {  
+
+	public  static Concept getDiscontinuedReason(DrugOrder dor){
```

Comment:
I would use `drugOrder` as the parameter name instead of `dor`.

Reply:
[Done](https://github.com/PIH/openmrs-module-rwandahivflowsheet/pull/2/commits/a6169460219bcfed598fc0a6f7555d9816327f3a)